src/main/java/seedu/address/model/FlashCardTestModel.java

```diff
@@ -0,0 +1,36 @@
+package seedu.address.model;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.ArrayList;
+
+import seedu.address.model.flashcard.FlashCard;
+
+//@@author keiteo
+/**
+ * Creates a test model to contain relevant flashcards to test users.
+ */
+public class FlashCardTestModel {
+
+    private ArrayList<FlashCard> testList;
+    private FlashCard currentFlashCard;
+
+    public FlashCardTestModel(ArrayList<FlashCard> testList) {
+        this.testList = testList;
+    }
+
+    public boolean isEmpty() {
+        return testList.isEmpty();
+    }
+
+    public String getQuestion() {
+        assert(!testList.isEmpty());
+        currentFlashCard = testList.remove(0);
```

Comment:
Removing the first element of an ArrayList is quite inefficient. You might want to use a LinkedList for this instead. Alternatively, you could keep the ArrayList but remove the last element instead of the first.